uniffi_meta/src/reader.rs

```diff
@@ -339,6 +341,7 @@ impl<'a> MetadataReader<'a> {
         Ok(ObjectMetadata {
             module_path: self.read_string()?,
             name: self.read_string()?,
+            remote: false, // only used when generating scaffolding from UDL
```

Comment:
These comments are a bit sad.  At some point, I think we should separate the metadata that is used to generate the scaffolding from the metadata used to generate the bindings.